.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "python.autoComplete.extraPaths": [
+        "/home/willy/ros_sonia_ws/devel/lib/python2.7/dist-packages",
+        "/opt/ros/melodic/lib/python2.7/dist-packages"
+    ],
+    "python.analysis.extraPaths": [
+        "/home/willy/ros_sonia_ws/devel/lib/python2.7/dist-packages",
+        "/opt/ros/melodic/lib/python2.7/dist-packages"
+    ]
+}
```

Comment:
This file should not be here, it will make problems on another computer.

Reply:
File deleted as requested.